build/constants.yaml

```diff
@@ -3,6 +3,7 @@
     environment-type: ORYX_ENV_TYPE # Currently exported only in the Visual Studio Online build image
     application-insights-instrumentation-key-env-var-name: ORYX_AI_INSTRUMENTATION_KEY
     app-service-app-name-env-var-name: APPSETTING_WEBSITE_SITE_NAME
+    app-service-pre-run-script-file-path-env-var-name: APPSETTING_PRE_RUN_SCRIPT_FILE_PATH
```

Comment:
Since we want this environment variable to be used by user to either provide command(s) or a script path, let's name it something like `PRE_RUN_COMMAND`. Also do not include the prefix `APPSETTING` since it is implicit. The setting `APPSETTING_WEBSITE_SITE_NAME` is from `AppService` actually which we use to log in some metrics.